selfdrive/ui/qt/maps/map.cc

```diff
@@ -155,14 +155,13 @@ void MapWindow::updateState(const UIState &s) {
     }
   }
 
-  if (sm.updated("navRoute") && sm["navRoute"].getNavRoute().getCoordinates().size()) {
```

Comment:
There may be a reason this was here: https://github.com/commaai/openpilot/pull/24352/files

I believe if you tap a destination without internet, it will popup with the invalid navRoute, correct?